docs/en/Remote-Logging.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: Remote Logging
+group: Features
+priority: 5
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+## Configure Alluxio
+Suppose the hostname of the log server is AlluxioLogServer, and the port is 45010.
+Refer to [Running Alluxio on a cluster](Running-Alluxio-on-a-Cluster.html) for instructions in deploying
+Alluxio in a cluster.
+
+By default, remote logging is not enabled. To enable Alluxio remote logging, you can either set a
+few environment variables or modify the JVM properties directly.
+
+This section describes these two different approaches step by step. There is no requirement on where
+the log server must run, as long as the other Alluxio servers have access to it. In our example, we
+run the log server on the same machine as the primary master. If you want to run the log server on
+another machine, you need to copy the Alluxio directory to the desired machine (with `rsync`
+for example) and start the log server there.
+
+### Enable Remote Logging with Environment Variables
+In ./conf/alluxio-env.sh, add the following lines:
+
+```bash
+ALLUXIO_LOGSERVER_HOSTNAME=AlluxioLogServer
+ALLUXIO_LOGSERVER_PORT=45010
+ALLUXIO_LOGSERVER_LOGS_DIR=/tmp/alluxio_remote_logs
+```
+
+### Enable Remote Logging with JVM Properties
```

Comment:
When should users use this approach instead of the Environment Variables approach?

Reply:
This approach is less intuitive than the environment variable approach, but I think some advanced users might want to know this possibility.

Reply:
If we give the user two ways to do something, the first thing they want to know is which one is right for them. We need to make it clear in the doc when to use which approach

Reply:
Done.